assets/src/stories-editor/blocks/amp-story-page/page-settings.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	PanelBody,
+	SelectControl,
+	RangeControl,
+} from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { getTotalAnimationDuration } from '../../helpers';
+
+const PageSettings = ( { autoAdvanceAfter, autoAdvanceAfterDuration, setAttributes, clientId } ) => {
+	let autoAdvanceAfterHelp;
+
+	if ( 'media' === autoAdvanceAfter ) {
```

Comment:
Comment explaining code.

Reply:
The component itself or specifically this condition here?

FWIW this is not newly introduced code, it just has been extracted into its own file.